server/api/router.go

```diff
@@ -22,6 +22,7 @@ import (
 	"github.com/gorilla/mux"
 	"github.com/pingcap/failpoint"
 	"github.com/tikv/pd/pkg/audit"
+	"github.com/tikv/pd/pkg/ratelimiter"
```

Comment:
https://github.com/tikv/pd/issues/4206
I think this issue also needs to be resolved, current the rate limiter has a license issue. IMO:
- do not rely on the `pkg/ratelimiter` in this PR
- fix the issue i mentioned before.

Reply:
> #4206 I think this issue also needs to be resolved, current the rate limiter has a license issue. IMO:
> 
> * do not rely on the `pkg/ratelimiter` in this PR
> * fix the issue i mentioned before.

 `pkg/ratelimiter` in this PR is just like an interface. It doesn't use `juju`.  
#4501 implements it by `golang.org/x/time/rate`.